scripts/ci/stats/merged_prs.py

```diff
@@ -98,42 +105,44 @@ def main():
             prj['reviewed_by_assignee'] = "na"
             prj['review_rule'] = "na"
 
-        # go through all assignees and check if anyone has approved and reset assignee to the one who approved
+        # go through all assignees and check if assignee is the submitter
         for assignee in pr.assignees:
-            if assignee.login in reviewers:
-                prj['assignee'] = assignee.login
-            elif assignee.login == pr.user.login:
-                prj['reviewed_by_assignee'] = "yes"
-
+            if  assignee.login == pr.user.login:
+                prj['review_rule'] = "yes"
```

Comment:
sorry, what's this rule supposed to be, I'm drawing a blank here? And in any case can't the check be performed as part of the earlier `for assignee in pr.assignees` loop?

Reply:
addressed